src/ggml-alloc.c

```diff
@@ -670,7 +678,10 @@ static void ggml_gallocr_alloc_graph_impl(ggml_gallocr_t galloc, struct ggml_cgr
     }
 }
 
+// Returns true on success, false otherwise
+// Check with reviewers: any cons to return a ggml_status?
```

Comment:
It would be ok to change the `gallocr` functions to return a `ggml_status`.

Reply:
ok, retouching that PR.